opm/simulators/wells/GasLiftSingleWellGeneric.cpp

```diff
@@ -666,6 +666,13 @@ getRateWithLimit_(Rate rate_type, const BasicRates& rates) const
     //   for why the rate was limited.
     std::optional<Rate> target_type;
 
+    Scalar rate2;
+    if (rate_type == Rate::oil) {
+        rate2 = getRate_(Rate::water, rates);
+    } else {
+        rate2 = getRate_(Rate::oil, rates);
+    }
+
```

Comment:
`rate2` is only really used with `Rate::oil` and `Rate::water` so perhaps a comment here to indicate that for future reference? Could also initialize `rate2 = 0.0` and change the `else` to `else if (rate_type == Rate::water)` to indicate this further, but I leave that up to you.

Reply:
I have updated accordingly.